profile/README.md

```diff
@@ -1,6 +1,60 @@
+
+### 👷 Platform Team
+
+Our Platform Team owns core components that serve the needs of our student and teacher teams (for example, authentication and user profile) and also has the mission of reaching students who we can't yet reach. This includes building integrations that connect us with schools that use different Learning Management Systems (LMS) and packaging our curriculum so that it can be used in remote schools who don't have access to reliable internet.
+
+### 🏗️ Infrastructure Team
+
+Our Infrastructure Team provides the backbone for the platform that supports our students and teachers. This includes our hosting infrastructure and related services (we use AWS), data management, security programs, developer and deployment services and much more. This team also owns our scaling and provisioning systems, which are critical during Hour of Code, where we invite every student in the world to complete an hour of coding with us.
+
+### Frequently Asked Questions
+
+#### 💻 What is Code.org's developer experience and tooling?
+
+Our developer experience may be different to what you’ve seen in your past roles. Working closely with our product team, we ship to production every weekday and we have a variety of cadences - sometimes you are writing code that goes live that day; other times you are on longer term projects - and while we do follow 2-weekly sprints, we do so in a lightweight way. We index towards getting the right things done the right way vs. taking a purist view of any particular methodology. [All our code](https://github.com/code-dot-org) is open source and we use JIRA and GitHub pull requests to track all of our work items. We also care a lot about quality and testing and are fortunate to have incredible support from [SauceLabs](https://saucelabs.com/), who help us run our suite of UI tests on various student devices (e.g., iPads) in their cloud device farm.
+
+#### 📟 What is the engineer on-call rotation at Code.org?
+
+Our 30+ engineering team shares on-call responsibilities. Each engineer covers a 24-hour shift and uses our escalation paths and playbooks to respond to site outages or other live-site issues. We are proud of our site’s reliability - on average, each engineer is only woken up once per year for issues that are typically quick to resolve. Instead, the "Developer of the Day" usually spends a portion of their workday managing the production deployment and triaging less urgent issues.
+
+#### 🏡 Are you a remote-friendly or work from home organization?
+
+Yes! In 2019, shortly before the pandemic, Code.org became a remote-first organization. Today, we have engineers in every US time zone and accommodate working hours accordingly. Most of our meetings happen between the hours of 10am and 3pm PT.
```

Comment:
Does 10am to 3pm still feel right? 9am to 2pm pacific seems like more of the norm these days but I am on a team that's split between east and west coast, so that might just be my experience.

Reply:
Code Tools is also 50/50 in Maine and Washington, so 9 AM to 2 PM is right for us, too. If this is a team-by-team thing, we could also avoid specifying the window in this section...